### ISSUE TITLE
How to save open3d.cpu.pybind.t.geometry.TriangleMesh object as .glb file?

### ISSUE THREAD

**Issue by arjung128** [isl-org/Open3D#5100]
### Checklist

- [X] I have searched for [similar issues](https://github.com/isl-org/Open3D/issues).
- [X] For Python issues, I have tested with the [latest development wheel](http://www.open3d.org/docs/latest/getting_started.html#development-version-pip).
- [X] I have checked the [release documentation](http://www.open3d.org/docs/release/) and the [latest documentation](http://www.open3d.org/docs/latest/) (for `master` branch).

### My Question

I have a `open3d.cpu.pybind.t.geometry.TriangleMesh` object. How can I save it as a `.glb` file?

o3d.io.write_triangle_mesh("test.glb", mesh) only seems to work for `open3d.cpu.pybind.geometry.TriangleMesh` objects.

**Comment by yuecideng** [comment]
Currently you can convert tensor based `TriangleMesh` to legacy `TriangleMesh` by calling `ToLegacy()`, then save it to `.glb` file.

**Comment by arjung128** [comment]
Thanks for your suggestion, but unfortunately this workaround does not work for me because I need to use the `mesh.triangle["colors"]` to color triangle faces (rather than vertices, as in the legacy version). When I do this conversion to legacy, the colors are not correctly represented in the legacy version (and thus in the `.glb` file)...

**Comment by yuecideng** [comment]
You are right. The Open3D team is working on extending the I/O of tensor based geometry ([GSoC](https://github.com/isl-org/Open3D/wiki/Google-Summer-of-Code-2022)). So maybe we can see this feature in next release...
